src/Raku/Actions.nqp

```diff
@@ -369,8 +391,15 @@ class Raku::Actions is HLL::Actions does Raku::CommonActions {
         my %OPTIONS  := %*OPTIONS;
         my $RESOLVER := $*R;
 
+        # Do any requested wrapping (-n or -p)
+        my $statement-list := $<statementlist>.ast;
+        if (my $add-print-topic := nqp::existskey(%OPTIONS,'p')) || nqp::existskey(%OPTIONS,'n') {
+            $statement-list.add-statement(print-topic()) if $add-print-topic;
```

Comment:
Won’t this print with ‘-n’ *or* ‘-p’ instead of just ‘-p’?

Reply:
I think you missed the postfix-if here

Reply:
Ah no, I missed that the bind to $add-print-topic is tighter than the ||.